reposerver/repository/testdata/invalid-k8s-manifest/invalid.json

```diff
@@ -0,0 +1 @@
+{"apiVersion":"","app":"argo-cd"}
```

Comment:
Could we add a few more test cases here? (`apiVersion` not being present, `kind` being present but not `apiVersion` and whatever other permutations you can think of?)

Reply:
ok, I will update the code.